products/db-knownprojects/python/__init__.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+from pathlib import Path
+from dotenv import load_dotenv
+from datetime import datetime
+
+_module_top_path = Path(__file__).resolve().parent
+_product_path = _module_top_path.parent
+_proj_root = _product_path.parent.parent
+
+# Make `dcpy` available
+sys.path.append(str(_proj_root))
+
+S3_SOURCE_BUCKET = "edm-private"
+S3_SOURCE_HOUSING_TEAM_DIR = "dcp_housing_team/db-knownprojects"
+
+DATA_PATH = _product_path / "data"
+RAW_DATA_PATH = DATA_PATH / "raw"
+PROCESSED_DATA_PATH = DATA_PATH / "processed"
+CORRECTIONS_DATA_PATH = DATA_PATH / "corrections"
+
+DCP_HOUSING_DATA_FILENAMES = {
+    "esd_projects": "2021.2.10 State Developments for Housing Pipeline.xlsx",
+    "edc_projects": "2022.11.18 EDC inputs for DCP housing projections.xlsx",
+    "edc_dcp_inputs": "edc_shapefile_20221118.zip",
+    "dcp_n_study": "nstudy_rezoning_commitments_shapefile_20221017.zip",
+    "dcp_n_study_future": "future_nstudy_shapefile_20221017.zip",
+    "dcp_n_study_projected": "PastNeighborhoodStudies_20221019.zip",
+    "hpd_rfp": "20221122_HPD_RFPs.xlsx",
+    "hpd_pc": "2022_11_23 DCP_SCA Pipeline.xlsx",
+    "dcp_planneradded": "dcp_planneradded_2023_03_21.csv",
+}
+DCP_HOUSING_CORRECTIONS_FILENAMES = {
+    "corrections_dob": "corrections_dob.csv",
+    "corrections_main": "corrections_main.csv",
+    "corrections_project": "corrections_project.csv",
+    "corrections_zap": "corrections_zap.csv",
+    "zap_record_ids": "zap_record_ids.csv",
+}
+
+
+# Today's date
+DATE = datetime.today().strftime("%Y-%m-%d")
+
+# Load environmental variables
+load_dotenv()
+BUILD_ENGINE = os.environ["BUILD_ENGINE"]
+
+# Create temporary output directories
+current_dir = os.getcwd()
+output_dir = f"{current_dir}/.output"
+
+if not os.path.isdir(output_dir):
```

Comment:
✂️ ?

Reply:
if this is referring to lines 55-57, agreed! and [there already is one](https://github.com/NYCPlanning/data-engineering/blob/fix-failing-builds/products/db-knownprojects/.gitignore) so I'll update it and drop lines 55-57

Reply:
even better, [the top-level .gitignore](https://github.com/NYCPlanning/data-engineering/blob/fix-failing-builds/.gitignore#L4) covers this!